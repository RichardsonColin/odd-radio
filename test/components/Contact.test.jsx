import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// React 16 Enzyme adapter
Enzyme.configure({ adapter: new Adapter() });

import Contact from '../../src/client/components/Contact';

describe('Contact', () => {

  const subject = <Contact />;

  it('matches snapshot', () => {
    expect(renderer.create(subject)).toMatchSnapshot();
  });
});