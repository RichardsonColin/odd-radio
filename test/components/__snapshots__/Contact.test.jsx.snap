// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact matches snapshot 1`] = `
<div
  className="container masthead-info"
>
  <div
    className="row center"
  >
    <h5>
      Contact Us
    </h5>
  </div>
  <div
    className="row"
  >
    <p>
      Got a suggestion for a station to add?
    </p>
    <p>
      Want us to take your station down?
    </p>
    <p>
      Get in touch with us by email: oddradioapp@gmail.com
    </p>
    <p>
      This is an open source project and we welcome additional contributions from the community.
    </p>
    <a
      href="https://github.com/johnniereg/odd-radio"
      target="blank"
    >
      <p>
        The code.
      </p>
    </a>
  </div>
</div>
`;
